src/borg/remote.py

```diff
@@ -56,6 +56,8 @@ class RepositoryServer:  # pragma: no cover
         'save_key',
         'load_key',
         'break_lock',
+        'get_free_nonce',
+        'commit_nonce_reservation'
```

Comment:
No, that won’t work. We need to be able to commit the reservation independently of normal repository commiting. And we need to be able to save the reservation without using the main encryption key.
So don’t think there is any way to include it into the normal chunk storage. And i don‘t think writing it into the config file would be any better (still needs new methods, it really doesn’t belong there). The semantics just don’t match with what we currently have.